app/bundles/CoreBundle/Tests/Twig/ThemesExtensionTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Mautic\CoreBundle\Tests\Twig;
+
+use Mautic\CoreBundle\Helper\CoreParametersHelper;
+use Mautic\CoreBundle\Twig\Extension\ThemesExtension;
+use PHPUnit\Framework\TestCase;
+
+class ThemesExtensionTest extends TestCase
+{
+    private CoreParametersHelper $coreParametersHelper;
```

Comment:
```suggestion
    private MockObject&CoreParametersHelper $coreParametersHelper;
```
This is a combination of PHPUNIT's MockObject and the actual method.